### DISCUSSION TITLE
[Projects Beta] Support adding default repo for project

### DISCUSSION THREAD

**Discussion by mofax** [community/community#8251] [Projects and Issues]
When converting an item into an issue, it would be nice to have the default repo, either pre-selected or placed at the top.

[alternatively] allow managing a project from the projects tab of the individual repo, in which case, the current repo is automatically selected.

**Comment by chamons** [comment]
I have nearly created issues in unrelated projects multiple times today due to this. A given board should at least remember the last issue created and default to that at the top of the list.

**Reply by nogginbox** [reply]
I've had the same issue. I'd like to see the option to [set allowed repos for a project](/github/feedback/discussions/14225). It could default to "All", but being able to restrict this would stop people accidentally creating issues in unrelated repos.

**Comment by jaumard** [comment]
also "add item" should use this default repo for searching item to add.

**Comment by alhuede-thncs** [comment]
I also think this is essential, as all my projects are created at the organization level however refer to specific repositories.

Accidentally creating an issue for an unrelated repository is a huge pain.

**Comment by cbaldry** [comment]
I think we just need some bulk update feature for this 🤔 
As a project manager update selected items to issue/repo/label etc

**Comment by bisdevres** [comment]
a big UP 🔝 for this one

**Comment by thorikawa** [comment]
I started using Porjects yesterday and am facing this problem.It is very tedious to select the repository each time, so I would be glad if there is some workaround.

**Comment by ctdesing** [comment]
Being able to create project items for multiple repos is a great feature, but there should be a select about setting a default repo, so it is in sync all the time.

**Comment by bchr02** [comment]
+1

**Comment by chris-kentik** [comment]
+1

**Comment by hayd** [comment]
This makes the Projects near-useless.

Projects seem to default to a random repository in my organization (not the one I linked too/created the project from).

Bizarre this isn't fixed after 2+ years.

**Comment by chrislind** [comment]
They don't fix this and are surprised when Projects isn't well used? Really.

**Comment by mshamaseen** [comment]
+1

**Comment by vc1492a** [comment]
+1

**Comment by compromyse** [comment]
+1

**Comment by usirg** [comment]
+1

very curious how this one is going to go - GitHub seems to either not see or ignore such easy-to-fix but very annoying ux issues

**Comment by dwolrdcojp** [comment]
We need this!

**Comment by linus-guestrix** [comment]
+1

**Comment by svaningelgem** [comment]
+1

**Comment by thorin-schiffer** [comment]
Users: associate projects with repositories
Microsoft: now you can congratulate your grandma on her birthday with GitHub! Watch our 213540925 tutorials and 72895709 texts and also our employees will visit your grandma and do some crocheting with her. Transgender grandpersons are also supported!

**Comment by jordanzdimirovic** [comment]
+1

**Comment by BrittanyTroast-NOAA** [comment]
Please add a select default repo for new issues so that projects help keep things organized instead of adding chaos when the wrong repo is chosen! A repo pops up anyways so might as well be the one you needl.

**Comment by HMKnapp** [comment]
+1. Not just a default repo where you can still click on the wrong one, but only show certain repos per project

**Comment by reececomo** [comment]
+1

We would happily switch from Jira/Notion to GitHub Projects if it had this one awesome feature ❤️ We just want to view Repository Issues in a kanban board and Projects seems the right way to do it, but its a little too clumsy right now.

**Comment by thinhlejax** [comment]
+1

**Comment by dallenbaldwin** [comment]
+1 

[this](https://github.com/orgs/community/discussions/60258) is relevant to this request
